### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by ebroder**

### CODE REVIEW

**Comment by zarvox** [comment]
I thought popper was the thing that powered the popovers on hover -- if you remove it, do we lose e.g. the related-puzzles popovers?

We seem to import from `@popperjs/core` in a couple places in imports/client/components/Tag.tsx

**Comment by zarvox** [comment]
Oh I see you commented that that's a different package.

We probably *should* have an explicit dep on popperjs if we're importing it in our source, rather than relying on it transitively.

**Comment by ebroder** [comment]
Turns out there's an eslint for that :)

File: .meteor/versions [hunk 1]
```diff
@@ -11,6 +11,7 @@ babel-runtime@1.5.0
 base64@1.0.12
 binary-heap@1.0.11
 boilerplate-generator@1.7.1
+bundle-visualizer@1.2.3
```
Comment (zarvox) [inline .meteor/versions:14]: Probably don't actually want to check this in.

**Comment by zarvox** [comment]
unintended `bundle-visualizer` dep aside, LGTM

File: .meteor/versions (hunk 1 above)
Comment (ebroder) [reply .meteor/versions]: Gah, damnit. Fixed

**Review by zarvox — APPROVED** [review]
Thanks for the cleanups!

**Pull request merged**